spoutbreeze-agent/src/main/java/org/spoutbreeze/agent/SpoutbreezeAgentApplication.java

```diff
@@ -36,7 +36,6 @@
 
 @SpringBootApplication
 @EnableScheduling
-@ComponentScan("module-service")
```

Comment:
Removed this because due to this, spring was not able to scan the entity or model classes.

Reply:
Thank you @himanshugoyal1065